master/usage/migrating-pools.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Migrating Active Pools
+---
+
+## About migrating active pools
+
+When using Calico-IPAM, each workload is assigned an address from the configured pools.
+You may want to change the pool being used on an active cluster for any of the following reasons:
+- You are migrating to a different pool with a larger cidr in order to accomodate more workloads.
+- You are transitioning from one cidr to a completely different cidr. The first cidr may have been chosen accidentally.
+
+### Purpose of this page
+
+Provide guidance on how to migrate off of a live Calico pool.
+
+## Prerequisites
+
+**Orchestrator Support**
+
+While calico supports changing the live address range, not all orchestrators do. 
+Be sure to consult the documentation for whichever installer you are using to ensure it supports changing it. 
+
+For example, in Kubernetes, you will also need to change:
+- kube-apiserver's `--pod-network-cidr`
+- kube-proxy & kube-controller-manager's `--cluster-cidr`
+
+**Application Availability Impact**
+
+This process wil require the recreation of all Calico-networked pods, which will have some impact on availability
+of your applications.
+
+**Consequences of deleting of an active Pool without following this migration procedure**
+
+Removing a pool actively being used will be bad. If the pool is removed before all workloads are moved, those workloads may experience networking difficulties, namely Calico will no longer NAT the traffic (if nat-outgoing is enabled) or properly encapsulate traffic (if using IP-IP mode).
+
+Removing an active pool without following the migration procedure will cause network connectivity disruptions in running 
+workloads. Namely, Calico will no longer NAT the traffic (if nat-outgoing is enabled) or properly encapsulate traffic (if using IP-IP mode).
+
+## Migrating off an active pool
+
+The basic process is as follows:
+
+1. Add a new pool.
+2. Disable the old pool.
```

Comment:
I think users will worry about a word like "disable", might be worth spelling out the impact wither here or in the more detailed section below `(this prevents new IPAM allocations from the old pool without affecting the networking of existing workloads)`